### ISSUE TITLE
pretrain loss

### ISSUE THREAD

**Issue by bokesyo** [QwenLM/Qwen#86]
Hi, i would like to know the final pretrain loss of Qwen-7B, just read the paper [arxiv](https://arxiv.org/abs/2308.01825) and curious about that, thank you!

**Comment by jklj077** [comment]
Hi, training losses are related to data mixture, vocabulary size, and so on. It should be used to meaningfully compare models. The paper you've mentioned explicitly states

> We should notice that pre-training losses correspond to different pre-training datasets and different tokenizers which means they could not be compared strictly (and we cannot use it to do any sort of regression directly).

Unfortunately, we cannot provide the final loss. However, if the observation from the paper generally holds, it should apply to Qwen as well.
